gossip/privdata/distributor.go

```diff
@@ -189,7 +189,7 @@ func (d *distributorImpl) computeDisseminationPlan(txID string,
 			logger.Debugf("Computing dissemination plan for collection [%s]", collectionName)
 			dPlan, err := d.disseminationPlanForMsg(colAP, colFilter, pvtDataMsg)
 			if err != nil {
-				return nil, errors.WithStack(err)
+				return nil, errors.Wrapf(err, "could not build private data dissemination plan for chaincode %s and collection %s", namespace, collectionName)
```

Comment:
same as other place "errors.WithMessage"

Reply:
This is a little dicey. Earlier, it was errors.WithStack... Ideally, we should use errors.New at the origin of the error and everywhere else either propagate the error or errors.WithMessage. This all of a sudden "errors.WithStack" at a random place does not leave with certainty what form of err is being received...